pkg/cedar/core/cedar_core.yml

```diff
@@ -0,0 +1,3111 @@
+swagger: "2.0"
+info:
+  description: "The CMS Enterprise Data and Analytics Repository (CEDAR) Core API's. \n These interfaces contain CRUD (where applicable) operations for Budget, Component, Contract, Data Center, Deployment, Exchange, Person, Role Stake Holder, System Detail, System Summary and User.\n"
+  version: 1.0.0
+  title: CEDAR Core API
+  termsOfService: https://www.cedardev.cms.gov/
+  contact:
+    email: EnterpriseArchitecture@cms.hhs.gov
+host: cdrprdapigw1.awscloud.cms.local:5555
+basePath: /gateway/CEDAR Core API/1.0.0
+tags:
+  - name: authority to operate
+    description: CMS authority to operate (ATO) data from CFACTS
+  - name: budget
+    description: CMS budget projects funding a particular system
+  - name: component
+    description: CMS components
+  - name: contract
+    description: CMS contracts funded by budget projects
+  - name: cost type
+    description: CMS cost types
+  - name: data center
+    description: Data centers hosting CMS systems
+  - name: deployment
+    description: CMS deployment of a system
+  - name: domain model
+    description: Domain heirarchy areas that categorize CMS and Federal agencies' assets
+  - name: enumeration
+    description: CMS system enumerations
+  - name: exchange
+    description: CMS data exchanges
+  - name: organization
+    description: CMS oganizations
+  - name: person
+    description: CMS people
+  - name: role
+    description: CEDAR role assignments and types
+  - name: stakeholder
+    description: CMS stakeholders
+  - name: support contact
+    description: CMS suport contacts
+  - name: system
+    description: CMS system information
+  - name: user
+    description: CEDAR system users
+schemes:
+  - http
```

Comment:
I had to change this to `https` and regenerate the client files to test successfully.

Reply:
After further experimentation, the "Connection refused" error I was seeing stopped occurring, which is why I thought this was necessary. CEDAR automatically redirects to HTTPS, so this change isn't needed.